examples/invalid/missing_name.json

```diff
@@ -0,0 +1,33 @@
+{
+    "@type": "ngff:Image",
+    "multiscales": [
+        {
+            "version": "0.3",
+            "datasets": [
+                {
+                    "path": "path/to/0"
+                },
+                {
+                    "path": 0
+                }
+            ],
+            "type": "gaussian",
+            "metadata": {
+                "method": "skimage.transform.pyramid_gaussian",
```

Comment:
Out of curiosity, since this is very Python focus, so if one is reading the file in Java for example a similar function will need to be found

Reply:
I think that's just documenting how the file was created. It doesn't affect how you'd read the data in Python or Java.

Reply:
I was thinking if we should follow a similar strategy for the rendering settings e.g. Fiji.salt&Pepper cc @dominikl